client/test-worker/turbo-resolver/resolver.spec.js

```diff
@@ -24,3 +24,29 @@ test('Resolver resolves vue2 deps', async t => {
   t.deepEqual(Object.keys(result.appDependencies), ['vue']);
   t.equal(Object.keys(result.resDependencies).length, 0);
 });
+
+test('Resolver resolves invalid deps', async t => {
+  const r = new Resolver(new NpmHttpRegistry());
+
+  try {
+    await r.resolve({
+      'aninvalidmodulename': '^2.0.0'
+    });
+    t.fail('must failed');
+  } catch (error) {
+    t.equal(error.message, 'Could not load npm registry for aninvalidmodulename: Not found');
```

Comment:
t.rejects expects a promise (you cannot add await before of the timing).

If you want to try:
```js
t.rejects(r.resolve({'...':'...'}), /^Could not load npm registry for aninvalidmodulename: Not found$/);
```

Reply:
Nice, thanks